client/src/components/EditProfile/Payment/Payment.tsx

```diff
@@ -0,0 +1,26 @@
+import { Elements } from '@stripe/react-stripe-js';
+import { loadStripe } from '@stripe/stripe-js';
+import CheckoutForm from './PaymentForm/CheckoutForm';
+
+// Make sure to call `loadStripe` outside of a component’s render to avoid
+// recreating the `Stripe` object on every render.
```

Comment:
Is this comment required here? Remove if nto.

Reply:
👍🏻Done